plugins/woocommerce-admin/client/activity-panel/activity-panel.js

```diff
@@ -160,8 +166,12 @@ export const ActivityPanel = ( { isEmbedded, query } ) => {
 			requestingTaskListOptions: ! hasFinishedResolution(
 				'getTaskLists'
 			),
-			setupTaskListComplete: getTaskList( 'setup' )?.isComplete,
+			setupTaskListComplete: setupList?.isComplete,
 			setupTaskListHidden: isSetupTaskListHidden,
+			setupTasksCount: setupList?.tasks.length,
```

Comment:
This should probably check for visible tasks only, right?  Not sure if we want to consider snoozed tasks or not, but probably don't want to include dismissed tasks in here.

```js
		setupList?.tasks.filter(
				( task ) =>
					! task.isDismissed &&
					( ! task.isSnoozed || task.snoozedUntil < Date.now() )
			) || [];
```

Reply:
Yup, this is true. I went ahead and started to create a `getVisibleTasks` selector in the onboarding data store for this, only to find that Lourens had already created [a utility function](https://github.com/woocommerce/woocommerce/blob/2bee19896388611d2f9962f2ea9962dfedbe4cb4/packages/js/data/src/onboarding/utils.ts#L9) a while back. I've implemented that here, as well as in the reminder bar logic.